src/testing/test-transpile.ts

```diff
@@ -15,7 +15,9 @@ export function transpile(input: string, opts: TranspileOptions = {}): Transpile
     style: null,
     styleImportData: 'queryparams',
     target: 'es2015', // default to es2015
-    transformAliasedImportPaths: parseStencilTranspilePaths(process.env.__STENCIL_TRANSPILE_PATHS__),
+    transformAliasedImportPaths: parseStencilTranspilePaths(
+      process.env.__STENCIL_TRANSPILE_PATHS__ as string,
+    ),
```

Comment:
I'm not sure we (safely) can assert this to be a string. From my understanding, we rely on `parseStencilTranspilePaths` to do the transformation for the value assigned to `transformAliasedImportPaths` - I would think `parseStencilTranspilePaths` should accept any of the types of `__STENCIL_TRANSPILE_PATHS__`. Currently, it's typed as:
https://github.com/ionic-team/stencil/blob/45718df9b8f50072c91f58c99163f56be068ff26/src/testing/test-transpile.ts#L38-L40
where the function signature assumes that we accept a `string`. Wouldn't it be safer (in that we don't require a type assertion) if we change the function signature itself?

```diff
- function parseStencilTranspilePaths(stencilTranspilePaths: string): boolean {
+ function parseStencilTranspilePaths(stencilTranspilePaths: string | undefined): boolean {
  return stencilTranspilePaths === 'true' ? true : false;
}
```

Reply:
sure, makes sense